### PR TITLE
Sample migration 7

### PR DESCRIPTION
**Pull request by kul3r4**

### CODE REVIEW

File: src/20_Components/amp-facebook-like.html [hunk 1]
```diff
@@ -11,20 +11,23 @@
...
+<!-- Import the `amp-facebook-like` component to add a like button-->
 <script async custom-element="amp-facebook-like" src="https://cdn.ampproject.org/v0/amp-facebook-like-0.1.js"></script>
 <link rel="canonical" href="<%host%>/components/amp-facebook-like/">
 <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
 <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 <style amp-custom>
+:root {
+  --space-2: 1rem;   /* 16px */
+}
 amp-facebook-like {
```
Comment (sebastianbenz) [inline src/20_Components/amp-facebook-like.html]: Can you scope it to this sample

Comment (sebastianbenz) [reply src/20_Components/amp-facebook-like.html]: Actually, I think we can just remove this.

File: src/20_Components/amp-facebook.html [hunk 2]
```diff
@@ -11,35 +11,37 @@
...
+    <!-- ## Basic usage -->
     <!-- The `amp-facebook` component only requires the URL of the facebook post... -->
     <amp-facebook width="552" height="303"
                     layout="responsive"
                     data-href="https://www.facebook.com/zuck/posts/10102593740125791">
     </amp-facebook>
 
-    <!-- ... or video -->
+    <!-- ## Video -->
+    <!--  -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-facebook.html]: Can you add a sentence here for consistency, e.g.: "Here is a sample for a facebook video embed."

Comment (kul3r4) [reply src/20_Components/amp-facebook.html]: done

File: src/20_Components/amp-facebook-like.html (hunk 1 above)
Comment (kul3r4) [reply src/20_Components/amp-facebook-like.html]: done

**Pull request merged**